build-push-image/action.yml

```diff
@@ -51,10 +51,25 @@ inputs:
 runs:
   using: composite
   steps:
+    - name: Set ECR Repository Name
+      id: set-ecr-repo
+      shell: bash
+      run: |
+        # Get the current branch name
+        branch_name="${GITHUB_REF##*/}"
+
+        # Check if the branch is not main, master, or a release branch
+        if [[ "$branch_name" != "main" && "$branch_name" != "master" && "$branch_name" != release* && "$branch_name" != feature/* ]]; then
+          echo "ECR repository name with feature suffix"
+          echo "ECR_REPOSITORY=${{ inputs.ecr_repository }}-feature" >> $GITHUB_ENV
```

Comment:
aren't we implementing feature as prefix here?

Reply:
yes i am making the change .. Thanks for pointing it out

Reply:
Done